### ISSUE TITLE
Create a prototype UI/dashboard for the testbed

### ISSUE THREAD

**Issue by jaeddy** [Sage-Bionetworks/synevalharness#77]
The current app setup provides a Swagger UI for the API, but it'd be nice to start drafting a more user-friendly dashboard like what @briandoconnor has mocked up [here](https://docs.google.com/document/d/1Cl3EZ5B1V-nxEeUCOJKyTHHJRDhknJyopdtjNcq_7jI/edit?pli=1#). Could potentially try out a React front end (serving as a client for the testbed service) — or maybe look for something simpler.

**Comment by jb-adams** [comment]
Hi @jaeddy and @briandoconnor, I've started working on this following the hackathon. I've implemented a simple React app UI loosely based off of Brian's mockup in the orchestrator documentation.

I've developed in a separate [repo](https://github.com/ga4gh/cloud-interop-ui) from this one, and have also written up a short [document](https://docs.google.com/document/d/13DIHDPSbpSeXVfwjIYmbn4BMiipPNrY5ZbBb03QMi1M), describing how to get started with running the app, as well as the core features.

Basically, the UI lists the available plugins and its properties (pulled from sample plugin JSON files). The UI also allows users to create new schedules from plugins. Each plugin expects a set of required input parameters (for example, a base url to a service or an object id). When the form to create a new test schedule is presented, the user must populate these input values.

Please have a look and let me know what you think. Based on your feedback I can start building out more features.

**Comment by jaeddy** [comment]
This looks awesome, @jb-adams! Thanks for taking the lead on creating a prototype — especially given that I don't know React. :) I'll play around some with the app this morning and let you know if I have any questions.
